cmip6_downscaling/methods/common/utils.py

```diff
@@ -29,7 +53,7 @@ def zmetadata_exists(path: UPath):
         return (UPath(path) / '.zmetadata').exists()
 
 
-def blocking_to_zarr(ds: xr.Dataset, target):
+def blocking_to_zarr(ds: xr.Dataset, target, validate: bool = True):
```

Comment:
should we be using this function to write data to disk throughout the entire package? it appears that in some places we are directly calling `to_zarr()`

Reply:
Seems like a good call!